encoding/proto/proto_benchmark_test.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func setupBenchmarkProtoCodecInputs(payloadBaseSize uint32) []proto.Message {
-	payloadBase := make([]byte, payloadBaseSize)
+	payloadBase := make([]byte, 0, payloadBaseSize)
```

Comment:
It seems to me that the intention of this function to create 5 protobufs of size approximately `payloadBaseSize`.

So the protobuf bodies consist of `payloadBaseSize` 0s + a suffix ("one", "two", "three" etc.). Setting the length to 0 would make that protobuf sizes independent of `payloadBaseSize` as they would contain only "one", "two", "three" etc.

Do you have a reason to believe the existing behaviour is incorrect?